src/wp-includes/kses.php

```diff
@@ -1080,12 +1092,48 @@ function _wp_kses_split_callback( $matches ) {
 function wp_kses_split2( $content, $allowed_html, $allowed_protocols ) {
 	$content = wp_kses_stripslashes( $content );
 
-	// It matched a ">" character.
+	/*
```

Comment:
Should `wp_kses_split2` also get an updated `@since`?

Reply:
Yes, great idea. Thanks for catching it!